tests/testthat/test-generate_data.R

```diff
@@ -486,15 +481,14 @@ test_that("genMarkov works.", {
 
   # startProb
   ## pk
-  set.seed(23456)
   gm2 <- genMarkov(n = nind, transMat = matr, chainLen = nchain, startProb = "0.65;0.25;0.05;0.05")
 
   prop <- gm1[, .N, keyby = .(id, state)]
   prop[, p := N / sum(N), keyby = id]
   prop[, t.p := rep(theoretical_p, max(id))]
   prop[, dif := abs(p - t.p)]
 
-  expect_true(all(prop[, dif < 0.02]))
+  expect_true(all(prop[, dif < 0.03]))
```

Comment:
Is the larger dif enough to work without a seed?

Reply:
Yes - I think I ran something like 1 million iterations and felt comfortable with that range. There's probably a better way to test this, but that was what I came up with.